### ISSUE TITLE
typo in default-extend.graphql

### ISSUE THREAD

**Issue by olivier-o** [APIs-guru/graphql-faker#81]
there is typo in this file: 
https://github.com/APIs-guru/graphql-faker/blob/5db15f66f13c0c69f7d69b3be004ffd0ba455f05/src/default-extend.graphql#L15
if no SDL file is specified the default-extend is used. resulting in an endless firing of request returning the following message 
```
{"errors":[{"message":"Unknown directive \"listLenght\"."}]}
```
`listLenght` should be replaced by `listLength` (typo is the `t`)

**Comment by IvanGoncharov** [comment]
@olivier-o Thanks for detail description 👍 
Fixed by @knidarkness in #82 and released in `in 2.0.0-rc.13` 📦
Can you please test it?

**Comment by olivier-o** [comment]
@IvanGoncharov , tested and working accordingly 👍
